manifest.json

```diff
@@ -51,6 +51,7 @@
         "contextMenus",
         "webRequest",
         "webRequestBlocking",
-        "*://*.duckduckgo.com/"
+        "*://*.duckduckgo.com/",
+        "notifications"
```

Comment:
I do not like it very much, but this `permission` is required -- otherwise we won't be able to trigger the popup/notification.